### ISSUE TITLE
docker-compose configuration out of date

### ISSUE THREAD

**Issue by matisk0** [simpledotorg/simple-server#2108]
Unable to run simple-server with current docker-compose config, which still based on ``` ruby:2.5.1 ```

```
$ docker-compose up
```

```
Your Ruby version is 2.5.1, but your Gemfile specified 2.6.6
ERROR: Service 'server' failed to build: The command '/bin/sh -c bundle _1.17.3_ install' returned a non-zero code: 18
```

After setting specified ruby 2.6.6 version, docker-compose setup almost finished except the following error:
```
server_1    | Seeding Facilities: |==========================================================|
server_1    | ⭐️  Seed complete! Elasped time less than a minute ⭐️
server_1    | rails aborted!
server_1    | ArgumentError: invalid uri scheme ''
server_1    | /usr/local/bundle/gems/redis-4.1.4/lib/redis/client.rb:439:in `_parse_options'
server_1    | /usr/local/bundle/gems/redis-4.1.4/lib/redis/client.rb:83:in `initialize'
server_1    | /usr/local/bundle/gems/redis-4.1.4/lib/redis.rb:44:in `new'
server_1    | /usr/local/bundle/gems/redis-4.1.4/lib/redis.rb:44:in `initialize'

server_1    | Tasks: TOP => db:refresh_materialized_views
server_1    | (See full trace by running task with --trace)
```

Seems like REDIS_URL isn't set properly or missing.

**Comment by benryder1988** [comment]
Thanks for raising the issue - I have some time this week and will look into it!

**Comment by benryder1988** [comment]
This should now have been fixed in the following pull request:
https://github.com/simpledotorg/simple-server/pull/2739

**Comment by rsanheim** [comment]
Fixed in #2739. Thanks!
